src/index.ts

```diff
@@ -172,20 +172,27 @@ export const submitTask = async (dataId: string, dataUserPk: string, wallet: any
   const nodeNames = exData.policy.names;
   const priceObj = JSON.parse(encData.price);
   const symbol = priceObj.symbol;
-  if (symbol !== 'AOCRED') {
-    throw new Error('Only support AOCRED now');
+  if ((symbol === 'AOCRED') || (symbol === 'wAR')) {
```

Comment:
The if statement can change to if ((symbol != 'AOCRED') && (symbol != 'wAR')) throw

Reply:
Refined.